app/assets/config/manifest.js

```diff
@@ -1,2 +1,2 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
+//= link_directory ../stylesheets .css
```

Comment:
Do not make changes like this. Add new line

Reply:
new line added